clang/lib/Driver/ToolChains/Darwin.cpp

```diff
@@ -2953,7 +2953,15 @@ static bool sdkSupportsBuiltinModules(
   case Darwin::MacOS:
     return SDKVersion >= VersionTuple(15U);
```

Comment:
zippered builds should fall under here too right?

Reply:
"zippered" builds can fall under either case, both are valid. I don't think the target variant is used for anything except tapi/ld?

Reply:
No, it's also encoded in object files. Can you add a testcase there too?

Reply:
You mean add a target variant to one of the lines? The Darwin driver never looks at the variant but we can add it anyway if you want. (Unzippered twin is very common too fwiw.)

Reply:
Unzippered invocation looks like a maccatalyst or macos invocation, right? So it's already covered. But yes, example invocation is in https://github.com/llvm/llvm-project/blob/main/clang/test/Driver/darwin-zippered-target-version.c#L2

Reply:
The extra test coverage is more for future-proofing. There is a decent amount of `TargetVariantTriple` logic happening in this file.

Reply:
Ok sure